mxcube3/__init__.py

```diff
@@ -158,6 +161,7 @@ def complete_initialization(app):
         except Exception:
             sys.excepthook(*sys.exc_info())
 
+        logging.getLogger("HWR").info("Mxcube3 server initialized, it took %.1f seconds" % (time.time() - t0))
```

Comment:
I guess this some kind of debug information, is it something that you are still using ? Does it mean that we have some un-deterministic delay when starting the server ?

If its meant to be permanent could we fix the casing of Mxcube3